tests/fms/kfto_kueue_sft_GPU_test.go

```diff
@@ -119,9 +156,20 @@ func runMultiGpuPytorchjob(t *testing.T, modelConfigFile string, numberOfGpus in
 	defer test.Client().Core().CoreV1().PersistentVolumeClaims(namespace).Delete(test.Ctx(), outputPvc.Name, metav1.DeleteOptions{})
 
 	// Create training PyTorch job
-	tuningJob := createAlpacaPyTorchJob(test, namespace, *config, numberOfGpus, outputPvc.Name, options...)
+	tuningJob := createAlpacaPyTorchJob(test, namespace, *config, numberOfGpus, outputPvc.Name, localQueue.Name, options...)
 	defer test.Client().Kubeflow().KubeflowV1().PyTorchJobs(namespace).Delete(test.Ctx(), tuningJob.Name, *metav1.NewDeleteOptions(0))
 
+	// Make sure the PyTorch job is admitted by Kueue
+	test.Eventually(KueueWorkloads(test, namespace), TestTimeoutLong).
+		Should(
+			And(
+				HaveLen(1),
+				ContainElement(
+					WithTransform(KueueWorkloadAdmitted, BeTrue()),
```

Comment:
May be it's good to add message in case of failure
```suggestion
					WithTransform(KueueWorkloadAdmitted, BeTrueBecause("Expected workload to be admitted by kueue")),
)
```